### DISCUSSION TITLE
full_node chia.full_node.full_node: WARNING  Block validation time: 5.88 seconds, pre_validation time: 0.40 seconds, cost: 4894000090, percent full: 44.491%

### DISCUSSION THREAD

**Discussion by farmingDCH** [Chia-Network/chia-blockchain#10386] [Q&A]
![image](https://user-images.githubusercontent.com/100044744/155255385-671edebb-1e31-4ee8-af37-8cb016d25d1f.png)

Will this affect the explosion block and how to deal with it?

**Comment by fengxulinlll** [comment]
请问这个日志从哪里调出来？
